demos/storybook/stories/user-menu/with-menu-header.stories.ts

```diff
@@ -0,0 +1,26 @@
+import { text } from '@storybook/addon-knobs';
+import { items } from './with-basic-config.stories';
+
+export const withMenuHeader = (): any => ({
+    template: `
+        <pxb-user-menu 
+            [avatarValue]="avatarValue" 
+            [menuTitle]="menuTitle" 
+            [menuSubtitle]="menuSubtitle"
+            [(open)]="open">
+            <mat-nav-list pxb-menu-body [style.paddingTop.px]="0">
+                <pxb-info-list-item *ngFor="let item of items" [dense]="true" (click)="open=false">
+                    <mat-icon pxb-icon>{{item.icon}}</mat-icon>
+                    <div pxb-title>{{item.title}}</div>
+                </pxb-info-list-item>
+            </mat-nav-list>
+    </pxb-user-menu> 
+    `,
+    props: {
+        open: false,
+        avatarValue: text('avatarValue', 'AV'),
```

Comment:
This story is the same as the full config. We can probably drop the avatarLabel knob here and just focus on the text part of the menu header (react only has the text knobs).

Reply:
Good catch, I've reverted the story name and created the menu items using mat-list-items instead of info-list-items to showcase the difference.

Reply:
I dropped the avatarLabel here and made the full config story more exciting.